_topic_maps/_topic_map.yml

```diff
@@ -4193,6 +4193,8 @@ Topics:
     File: virt-using-vtpm-devices
   - Name: Managing virtual machines with OpenShift Pipelines
     File: virt-managing-vms-openshift-pipelines
+  - Name: Configuring higher VM workload density
```

Comment:
Please follow these steps to determine if a redirect is required:

[link](https://docs.google.com/document/d/1fLMpK4bqthtFlCwA36yeo2cI-pExFpX76VMnSi_In6Q/edit#bookmark=id.xb255koulgqx)

Reply:
@dfitzmau  But these are my new files that I created in another PR. I just noticed that I had placed them not where I wanted, by mistake.

Reply:
Ah, OK. You are the master of their destiny so I guess you won't trip up anything here.